lib/utils.js

```diff
@@ -191,18 +322,104 @@ const getUsernamesFromText = (text) => {
   return usernames;
 };
 
+/**
+ * This function gets all the open issues in a repository.
+ * @param {import('probot').Context} context
+ */
+const getAllOpenIssues = async (context) => {
+  /**
+   * @type {import('probot').Octokit.IssuesListResponse}
+   */
+  const openIssues = [];
+  const maxAmountPerPage = 100;
+  let currentAmountFetched = 0;
+  let currentPage = 1;
+  do {
+    // This function returns both pull requests and issues.
+    const issuesResponse = await context.github.issues.listForRepo(
+      context.repo({
+        per_page: maxAmountPerPage,
+        state: 'open',
+        page: currentPage,
+      })
+    );
+
+    openIssues.push(...issuesResponse.data);
+    currentAmountFetched = issuesResponse.data.length;
+    currentPage++;
+  } while (currentAmountFetched === maxAmountPerPage);
+
+  // Pull requests are also returned from the api, hence we need to filter
+  // them out.
+  return openIssues.filter((issue) => 'pull_request' in issue === false);
+};
+
+/**
+ * This function gets all the project cards for a repository.
+ * @param {import('probot').Context} context
+ */
+const getAllProjectCards = async (context) => {
+  const { data: allProjects } = await context.github.projects.listForRepo(
```

Comment:
Please make sure line lengths are correct, throughout the PR.

Reply:
Yes.
This has 75 chars.